index.d.ts

```diff
@@ -47,7 +47,7 @@ declare module 'country-codes-list' {
   export function filter(
     countryProperty: CountryProperty,
     value: string,
-  ): CountryData[]
```

Comment:
Returning `CountryData[]` is actually correct since the filter function returns an array (see https://github.com/LucianoGanga/country-codes-list/blob/master/index.js#L24-L26)